src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -1012,6 +1012,26 @@ ss::future<response_ptr> describe_configs_handler::handle(
         }
     }
 
+    // If unable to audit, remove any trace of the response data
+    if (!ctx.audit()) {
```

Comment:
Why is this at the bottom of the method ? I'd imagine it can be done before anything occurs

Reply:
I was attempting to not modify this handler too much.  The handler will walk through each requested resource and check to see if the connection is authorized to view the resource.  If it was, it populates the response, else it skips the request.

It does not appear to modify state of the cluster (or am I wrong?) so I don't bother performing an audit call until after all the resource checks are performed.

Reply:
ok i guess it would be more code to move this at the top of the method then. Yeah I don't think this handler does any state modification so if thats the case, fine by me